meta/ibm_zos_core_meta.yml

```diff
@@ -1,5 +1,5 @@
 name: ibm_zos_core
-version: "1.5.0"
+version: meta/ibm_zos_core_meta.yml6.0-beta.1"
```

Comment:
Why this version name is meta/ibm_zos_core_meta.yml6.0-beta.1? looks like a typo